nextflow.config

```diff
@@ -51,52 +48,53 @@ params {
   spectrum_batch_size = 500
   variable_mods = 'Oxidation (M)'
   tracedir = "${params.outdir}/pipeline_info"
+
+  // Options: AWSBatch
+  clusterOptions = false
+  awsqueue = false
+  awsregion = 'eu-west-1'
+
+  // Options: Custom config
+  config_profile_description = false
+  config_profile_contact = false
+  config_profile_url = false
+
+  custom_config_version = 'master'
+  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
 }
 
+// Container slug. Stable releases should specify release tag!
+// Developmental code should specify :dev
+process.container = 'nfcore/mhcquant:1.2.6'
+
+// Load base.config by default for all pipelines
+includeConfig 'conf/base.config'
+
+// Load nf-core custom profiles from different Institutions
+try {
+  includeConfig "${params.custom_config_base}/nfcore_custom.config"
+} catch (Exception e) {
+  System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
+}
 
 profiles {
+  awsbatch { includeConfig 'conf/awsbatch.config' }
+  debug { process.beforeScript = 'echo $HOSTNAME' }
   docker {
     docker.enabled = true
-    process.container = params.container
```

Comment:
hmm.. , did this cause any problems or why remove it?

Reply:
Yes.
This is discouraged now and linting was complaining.
See https://github.com/nf-core/mhcquant/pull/58